lib/basevm.py

```diff
@@ -1,14 +1,13 @@
 import logging
+from typing import TYPE_CHECKING, Any, Literal, Optional, overload
 
-from typing import Any, Literal, Optional, overload, TYPE_CHECKING
-
-import lib.commands as commands
 if TYPE_CHECKING:
     import lib.host
 
 from lib.common import _param_add, _param_clear, _param_get, _param_remove, _param_set
 from lib.sr import SR
 
+
```

Comment:
Can ruff be made to reorder imports without enforcing these two lines? So many modified files...